ci/kubetest/test_clickhouse_backup.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import time
+
+from helpers.clickhouse import get_clickhouse_cluster_service_spec
+from helpers.utils import cleanup_k8s, cleanup_helm, helm_install, wait_for_pods_to_be_ready, install_chart
+from helpers.utils import (
+    NAMESPACE,
+    cleanup_helm,
+    cleanup_k8s,
+    exec_subprocess,
+    install_chart,
+    install_custom_resources,
+    merge_yaml,
+    wait_for_pods_to_be_ready,
+)
+
+
+
+from typing import Optional
+
+VALUES_WITH_BACKUP = """
+clickhouse:
+  backup:
+    enabled: true
+    backup_schedule: "*/2 * * * *"
```

Comment:
Why `*/2`? To speed up the test can we just put it every minute so it will kick within 60 seconds from the installation at max?

Reply:
sometimes clickhouse requires 80 seconds to start on slow machine, so I don't think to have it as 60 is good idea.